test/acceptance/framework/consul/consul_cluster.go

```diff
@@ -70,10 +70,18 @@ func NewHelmCluster(
 
 	logger := terratestLogger.New(logger.TestLogger{})
 
+	// Wait up to 15 min for K8s resources to be in a ready state. Increasing this
+	// from the default of 5 min could help with flakiness in environments like
+	// AKS where volumes take a long time to mount.
+	extraArgs := map[string][]string{
+		"install": {"--timeout", "15m", "--wait"},
```

Comment:
I don't think we should use `--wait` because we already have `WaitForAllPodsToBeReady` that gives us better logging about which pods aren't ready.

Reply:
Yeah that makes sense, thanks!